src/OmniSharp.MSBuild/ProjectFile/PackageReference.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using NuGet.Packaging.Core;
+
+namespace OmniSharp.MSBuild.ProjectFile
+{
+    public class PackageReference : IEquatable<PackageReference>
```

Comment:
Possibly. I *believe* these are used by NuGet to handle VS package restore, which is a much bigger thing than what we're doing. I just need to keep a list of the PackageReferences items computed by MSBuild so that I can tell if changes occurred to the project file. This is before NuGet is really involved.

Note that this class is not exactly the same. It has at least one piece of additional metadata that is provided via MSBuild targets.